### PR TITLE
transporterSS

### PR DESCRIPTION
**Pull request by gsyoyo07**

### CODE REVIEW

File: src/main/java/frc/robot/constants/RobotComponents.java [hunk 1]
```diff
@@ -95,4 +95,8 @@
                 CAN.Swerve.REAR_RIGHT_DRIVE_MOTOR_ID, DRIVE_MOTOR_CONFIG.withPID(
                 LOCAL_SWERVE_MODULES.rearRightModuleConstants.driveCoefs));
     }
+    protected static class TransporterComponents{
+        public static final TrigonTalonSRX MOTOR = SwerveComponents.FRONT_RIGHT_ENCODER;
```
Comment (Mboesn) [inline src/main/java/frc/robot/constants/RobotComponents.java:99]: shouldn't swerve take from transporter due to motorconfig? if not have a motor config effect the motor

File: src/main/java/frc/robot/subsystems/transporter/TransporterSS.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    public void setPower(double power){
+        motor.set(power);
+    }
+
+    @Override
+    public void overriddenMove(double power) {
+        setPower(power);
+    }
```
Comment (Mboesn) [inline src/main/java/frc/robot/subsystems/transporter/TransporterSS.java:22]: remove

**Review by Mboesn — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/constants/RobotComponents.java (hunk 1 above)
Comment (ArielSegal1) [reply src/main/java/frc/robot/constants/RobotComponents.java:99]: This is the way we did it for the rest... it should be fine... -Natan

File: src/main/java/frc/robot/subsystems/transporter/TransporterSS.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+public class TransporterSS extends OverridableSubsystem {
+    private final TrigonTalonSRX motor;
+
+    public TransporterSS() {
+        motor = TransporterConstants.MOTOR;
+    }
+
+    public void setPower(double power){
+        motor.set(power);
+    }
```
[the remark this answers is not in the document]
Comment (laviRZ) [reply src/main/java/frc/robot/subsystems/transporter/TransporterSS.java]: .

**Pull request merged**